fix VirtualBox-5.0.4/src/VBox/ExtPacks/VNC/VBoxVNC.cpp:339:30: error: invalid conversion from 'rfbNewClientAction (__attribute__((regparm(0), cdecl)) *)(rfbClientPtr) {aka rfbNewClientAction (__attribute__((regparm(0), cdecl)) *)(_rfbClientRec*)}' to 'rfbNewClientHookPtr {aka rfbNewClientAction (*)(_rfbClientRec*)}' [-fpermissive]
error for 32-bit build

See:
https://www.virtualbox.org/ticket/14564

--- VirtualBox-5.0.4/src/VBox/ExtPacks/VNC/VBoxVNC.cpp
+++ VirtualBox-5.0.4/src/VBox/ExtPacks/VNC/VBoxVNC.cpp
@@ -104,6 +104,6 @@
     unsigned char *mFrameBuffer;
     uint32_t uClients;
-    static DECLCALLBACK(enum rfbNewClientAction) rfbNewClientEvent(rfbClientPtr cl);
-    static DECLCALLBACK(void) vncMouseEvent(int buttonMask, int x, int y, rfbClientPtr cl);
+    static enum rfbNewClientAction rfbNewClientEvent(rfbClientPtr cl);
+    static void vncMouseEvent(int buttonMask, int x, int y, rfbClientPtr cl);
     static void vncKeyboardEvent(rfbBool down, rfbKeySym keySym, rfbClientPtr cl);
     static void clientGoneHook(rfbClientPtr cl);
